Makefile.windows

```diff
@@ -16,6 +16,8 @@
 #Windows specific settings.
 WHALE = "+"
 ONI = "-"
+CONTAINERD_SOCK ?= npipe://./pipe/containerd-containerd
```

Comment:
Wouldn't this interfere with a real containerd running on the same host? This is only used for the CRI test, I assume, so it probably should have a name that keeps it specific to those tests, assuming they run their own containerd to test against.

Perhaps it'd be better to set up in script/utils/test.sh where the Linux version of this variable is set, so they're visible together.

Reply:
Done